test/beast/core/basic_stream.cpp

```diff
@@ -591,9 +605,10 @@ class basic_stream_test
                 s.socket().connect(srv.local_endpoint());
                 s.expires_after(std::chrono::seconds(0));
                 s.async_read_some(mb,
-                    [](error_code, std::size_t)
-                    {
-                    });
+                                  [](
+                                      error_code,
+                                      std::size_t) {
+                                  });
```

Comment:
I don't indent like this and I dont think you should either it just widens the code needlessly